### PR TITLE
docs(ls-nr-intgr): add data store config and recipe for ls and nr

### PR DESCRIPTION
**Pull request by adnanrahic**
This PR adds docs for the Lightstep and New Relic integrations.

## Changes

-

## Fixes

-

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

**Review by xoscar — APPROVED** [review]
LGTM!

File: docs/docs/configuration/connecting-to-data-stores/lightstep.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# Lightstep
+
+If you want to use [Lightstep](https://lightstep.com/) as the trace data store, you'll configure the OpenTelemetry Collector to receive traces from your system and then send them to both Tracetest and Lightstep. And, you don't have to change your existing pipelines to do so.
+
+:::note
+It is important to notice that this relies on the [tailsampling](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/tailsamplingprocessor) processor, which, at the moment, is only available in the [contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib/) version of the collector.
```
Comment (jorgeepc) [inline docs/docs/configuration/connecting-to-data-stores/lightstep.md:6]: I think we removed the `tailsampling` support https://github.com/kubeshop/tracetest/pull/1806 Do we want to remove this note?

File: docs/docs/configuration/connecting-to-data-stores/new-relic.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
+# New Relic
+
+If you want to use [New Relic](https://newrelic.com/) as the trace data store, you'll configure the OpenTelemetry Collector to receive traces from your system and then send them to both Tracetest and New Relic. And, you don't have to change your existing pipelines to do so.
+
+:::note
+It is important to notice that this relies on the [tailsampling](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/tailsamplingprocessor) processor, which, at the moment, is only available in the [contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib/) version of the collector.
```
Comment (jorgeepc) [inline docs/docs/configuration/connecting-to-data-stores/new-relic.md:6]: same here

**Review by kdhamric — APPROVED** [review]

**Review by mathnogueira — APPROVED** [review]

File: docs/docs/configuration/connecting-to-data-stores/new-relic.md (hunk 2 above)
Comment (adnanrahic) [reply docs/docs/configuration/connecting-to-data-stores/new-relic.md]: Damn. Missed this one. Removed it.

**Pull request merged**